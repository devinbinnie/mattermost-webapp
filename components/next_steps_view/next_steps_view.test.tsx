// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import React from 'react';
import {shallow} from 'enzyme';

import NextStepsView from 'components/next_steps_view/next_steps_view';

describe('components/next_steps_view', () => {
    const baseProps = {
        currentUserId: 'user_id',
        preferences: [],
        skuName: '',
        actions: {
            setShowNextStepsView: jest.fn(),
            savePreferences: jest.fn(),
        },
    };

    test('should match snapshot', () => {
        const wrapper = shallow(
            <NextStepsView {...baseProps}/>,
        );

        expect(wrapper).toMatchSnapshot();
    });
});
