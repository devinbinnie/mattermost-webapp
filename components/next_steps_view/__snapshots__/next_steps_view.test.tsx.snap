// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/next_steps_view should match snapshot 1`] = `
<section
  className="app__content NextStepsView"
  id="app-content"
>
  <header
    className="NextStepsView__header"
  >
    <div
      className="NextStepsView__header-headerText"
    >
      <h1
        className="NextStepsView__header-headerTopText"
      >
        <FormattedMessage
          defaultMessage="Welcome to Mattermost"
          id="next_steps_view.welcomeToMattermost"
          values={Object {}}
        />
      </h1>
      <h2
        className="NextStepsView__header-headerBottomText"
      >
        <FormattedMessage
          defaultMessage="Here are some recommended next steps to help you get started"
          id="next_steps_view.hereAreSomeNextSteps"
          values={Object {}}
        />
      </h2>
    </div>
    <div
      className="NextStepsView__header-logo"
    >
      <img
        src={null}
      />
    </div>
  </header>
  <div
    className="NextStepsView__body"
  >
    <div
      className="NextStepsView__body-main"
    >
      <Accordion
        defaultExpandedKey="Card_1"
      >
        <Component />
      </Accordion>
    </div>
    <div
      className="NextStepsView__body-graphic"
    />
  </div>
</section>
`;
